modules/aruco/misc/pattern_generator/MarkerPrinter.py

```diff
@@ -0,0 +1,702 @@
+import numpy as np
+from PIL import Image
+import io
+import warnings
+import os
+import cairo
+import math
+from cairosvg import svg2png
+import tempfile
+
+def WriteArucoDict(filePath = "data.txt"):
```

Comment:
Ok~ I will add command line interface, and remove build-in data.txt

Reply:
Well, built-in data.txt can be useful for people who has no _cv2_ module, e.g. C++ or Java users. But current format is too wasteful - 390 KB, I believe it can be made more efficient, e.g. by packing all 0s and 1s to bitfields. Maybe even compressing with standard python algorithms (zip, gzip). We can investigate these possibilities later, after this PR is merged.

Reply:
Yes, I will find out where aruco module cache those precomputed dictionary  data, and reuse the. rather than duplicate them again in text

Reply:
I find them... They are already write in aruco/src/predefined_dictionaries.hpp and aruco/src/predefined_dictionaries_apriltag.hpp

Reply:
By the way, some of them are duplicated, EX: DICT_4X4_50_DATA ~ DICT_4X4_250_DATA are aready in DICT_4X4_1000_DATA, so basically only save DICT_4X4_1000_DATA is fine

Reply:
Now the data is save as npz (only save byteList, not marker bits).... only 146 KB... I think it is hard to be more smaller